### ISSUE TITLE
[Problem] Notif Area should truncate messages.

### ISSUE THREAD

**Issue by PaddleStroke** [FreeCAD/FreeCAD#10018]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

0.21 (Development)

### Full version info

```shell
0.21
```

### Subproject(s) affected?

Core

### Problem description

To avoid this kind of situation : 
![image](https://github.com/FreeCAD/FreeCAD/assets/19984177/6a5501d7-addf-47c5-a271-faca223888f9)

Where the notification area show giant and totally useless errors, I think the notification area should put a max size to the error message. The full sized message being visible in the report view.

So we could truncate the string to around 1 or 2 lines, then finish by '...' if it's longer than that.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by chennes** [comment]
I agree. Is it possible to make it clickable, and show the full text in a dialog or something?

**Comment by PaddleStroke** [comment]
There's already the button that opens the notification area view where we can see the past messages. In this we can elt the full message ?
By the way, I feel that this view is a bit redundant with the report view. But that's another discussion.

**Comment by maxwxyz** [comment]
* related: https://github.com/FreeCAD/FreeCAD/issues/10725

**Comment by luzpaz** [comment]
This seems like it can be closed, right ?

**Comment by PaddleStroke** [comment]
No it's still

Just made a test with a long message and it's still showing all : 

<img width="1553" height="1149" alt="Image" src="https://github.com/user-attachments/assets/2e813289-c4b6-4c2d-a951-8bc2704b87f4" />

**Comment by luzpaz** [comment]
@chennes maybe it's a good moment to start prioritizing the complaint session bugs in time for the next complaint session in 2026 ?
